docs/integrations/destinations/mssql-v2.md

```diff
@@ -12,10 +12,11 @@ This connector is in early access, and SHOULD NOT be used for production workloa
   <summary>Expand to review</summary>
 
 | Version | Date       | Pull Request                                               | Subject        |
-|:--------|:-----------| :--------------------------------------------------------- |:---------------|
-| 0.1.3 | 2025-01-24 | [52096](https://github.com/airbytehq/airbyte/pull/52096) | Release candidate |
-| 0.1.2 | 2025-01-10 | [51508](https://github.com/airbytehq/airbyte/pull/51508) | Use a non root base image |
-| 0.1.1 | 2024-12-18 | [49870](https://github.com/airbytehq/airbyte/pull/49870) | Use a base image: airbyte/java-connector-base:1.0.0 |
-| 0.1.0   | 2024-12-16 | [\#49460](https://github.com/airbytehq/airbyte/pull/49460)   | Initial commit |
+|:--------|:-----------|:-----------------------------------------------------------|:---------------|
+| 0.1.4   | 2025-02-04 | [52704](https://github.com/airbytehq/airbyte/pull/52704)   | Release candidate |
```

Comment:
Nit:  Should we mark this as "Release candidate 2"?

Reply:
updated with a message "slightly" more descriptive.